lib/drawers/publish-page.vue

```diff
@@ -326,6 +328,7 @@
       },
       clearScheduleForm() {
         this.dateValue = null;
+        this.timeValue = null;
```

Comment:
This property needs to be cleared too in order to reflect the changes in the UI.